kip-0017.md

```diff
@@ -0,0 +1,489 @@
+    1.1. `chains`: `Array<string>` - an array of [Chain IDs](#wallet-connect-chain-ids) requested.\
+    1.2. `methods`: `Array<string>` - an array of [Methods](#walletconnect-methods) requested.\
+    1.3. `events`: `Array<string>` - an array of [Events](#walletconnect-events) requested.
+
+### WalletConnect Settlement Response
+
+The response for the Proposal contains a `namespaces` object with the following properties:
+
+1. `namespaces`: `Object`\
+    1.1. `accounts`: `Array<string>` - an array of [Accounts](#wallet-connect-accounts) availble for signing.\
+    1.2. `methods`: `Array<string>` - an array of [Methods](#walletconnect-methods) available.\
+    1.3. `events`: `Array<string>` - an array of [Events](#walletconnect-events) available.
+
+## Wallet Connect Chain IDs
+
+Chain IDs are Wallet Connect’s way of identifying specific blockchains within an ecosystem.
+
+They are defined by the [CAIP-2 standard](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md) and are formatted as:
+
+```
+    <namespace> + ":" + <reference>
+```
+- `<namespace>`: `string` - always defined as "kadena".
+- `<reference>`: `string` - a unique network ID within the blockchain ecosystem (e.g. `testnet04`, `mainnet01`).
+
+These are valid Chain IDs:
+
+- `kadena:mainnet01` - Kadena mainnet
+- `kadena:testnet04` - Current Kadena testnet
+- `kadena:development` - Kadena devnet
+
+> **NB**: `Chain IDs`, as defined here, are semantically specific to
+> WalletConnect and are not equivalent to a Kadena "Chain ID" (e.g. chain 1).
+> They are more closely related to Kadena's `networkId` field.
+> We are displaying Kadena `chainId` information in the [`kadena_getAccounts_v1`](#kadena_getAccounts_v1) method.
+
+
+## Wallet Connect Accounts
+
+Accounts are Wallet Connect's way of specifying control over signing privileges.
+
+They are defined by the [CAIP-10 standard](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md), and are formatted as:
+```
+    <chain_id> + ":" + <account_address>
+```
+- `<chain_id>`: `string` - the Wallet Connect [Chain ID](#wallet-connect-chain-ids).
+- `<account_address>`: `string` - the public key available for signing by the wallet.
+
+```jsonc
+// Wallet Connect Account Example
+
+"kadena:mainnet01:38298612cc2d5e841a232bd08413aa5304f9ef3251575ee182345abc3807dd89"
+```
+
+> **NB**: `accounts` is semantically specific to WalletConnect. This does not
+> represent an account in a Kadena fungible contract (e.g. `coin`).
+> Read the motivation for this in [Accounts vs. Public Keys](#accounts-vs-public-keys).
+
+Some users will want to know which account names in a Kadena fungible contract are associated with the public keys returned by Wallet Connect.
+
+These users have two options:
+- Some account names can be derived from the public key (e.g. `k:` accounts that are defined as `k:<public_key>`).
+- For all other cases, the [`kadena_getAccounts_v1`](#kadena_getAccounts_v1)
+method can be called.
```

Comment:
Adding "Kadena" in front of accounts for some clarity.

```suggestion
After the initial pairing with the wallet via WalletConnect, the Kadena accounts relevant for each public key can be requested by calling [getAccounts](#kadena_getaccounts_v1).

The use of account names that can be derived from the public key (i.e. `k:`-accounts that are defined as `k:<public_key>`), is **highly discouraged** as implementing this will limit the end-users flexibility in which account to use, thus making dApps less generic.
```